index.html

```diff
@@ -184,9 +184,9 @@ <h3>
       <span its-locale-filter-list="zh-hant" lang="zh-hant">目的</span>
     </h3>
 
-    <p its-locale-filter-list="en" lang="en">Each cultural community has its own language, script, and writing system. The transfer of each and every writing system into cyberspace is a task of utmost importance for information and communication technology.</p>
-    <p its-locale-filter-list="zh-hans" lang="zh-hans">每一个文化群体都拥有独自的语言、文字、书写系统。将个别书写系统在虚拟空间再现，对文化资产的承继而言，是信息传播技术的重要责任。</p>
-    <p its-locale-filter-list="zh-hant" lang="zh-hant">每一個文化群體都擁有獨自的語言、文字、書寫系統。將個別書寫系統在虛擬空間再現，對文化資產的承繼而言，是資訊傳播技術的重要責任。</p>
+    <p its-locale-filter-list="en" lang="en">The transfer of each and every writing system into digital world is an important responsibility of information communication technology, which plays an important role in the generation, safeguarding, maintenance and re-creation of cultural assets.</p>
```

Comment:
This doesn't read as well, for me.  You need 'the' before 'digital world', but also does 'which' refer to the transfer or to the information communication technology?  'Information communication technology' sounds a bit odd – the original 'information and communication technology' sounded better. Does the technology bear a responsibility, or the people developing that technology? hth

Reply:
Agreed on the first two points, I think a better revision of this sentence could be:
"The transfer of each and every writing system into the digital world is an important responsibility of information and communication technology. It plays an important role in the generation, safeguarding, maintenance and re-creation of cultural assets."
The Chinese text reads that the technology bears the responsibility, and it can be understood that technology is ultimately developed by people, so it's probably fine to not explicitly state that.